src/app/mail/mail.component.html

```diff
@@ -3,7 +3,7 @@
 <div class="autoresponder-banner row ml-lg-5" *ngIf="autoresponder_status">
     <p>Autoresponder is active:
         <a (click)="endAutoResponder()" translate="settings.end_now">End now</a>&nbsp;-
-        <a [routerLink]="'./settings'" translate="mail_header.settings"> Settings</a>
+        <a [routerLink]="'./settings/auto'" translate="mail_header.settings"> Settings</a>
```

Comment:
the url param should be a proper name, change `auto` to `forwarding-and-auto-responder`